src/layer/form-layout/index.md

```diff
@@ -0,0 +1,103 @@
+![Mergin Maps Input form with tabs and groups](./input_forms_layout2.jpg "Mergin Maps Input form with tabs and groups")
+
+
+## Show and hide fields depending on a field value
+Conditional visibility can be applied to groups and tabs, meaning they will be displayed or hidden depending on the value of a field.
+
+We can set the visibility of **Location** tab depending on the value of the **survey** field:
+  - If **survey** is set to **False**, the **Location** tab will not appear in the form,
+  - If **survey** is set to **True**, the **Location** tab will show up in the form.
+
+1. Select the **Location** tab in the **Form Layout** column
+2. Check the **Control Visibility by Expression** option
+3. Type the following expression in the box below: `"survey" IS TRUE`
+4. Background colour can be set in the **Style** tab.
+![QGIS form control visibility by expression](./qgis_forms_layout_visibility.jpg "QGIS form control visibility by expression")
+
+## Display instructions in the form using Text and HTML widget
+Sometimes, you may want to include instructions or tips for surveyors. This can be achieved by using the Text or HTML widgets. These notes may also display [expressions and field values](#using-expressions-in-text-and-html-widgets), which can be useful.
+
+These widgets can be found in **Available Widgets** in the **Other Widgets** section when using the [Drag and Drop Designer](#qgis-drag-and-drop-designer).
+![QGIS HTML and Text Widget](./qgis-form-html-text-widget.jpg "QGIS HTML and Text Widget")
+
+To configure the **Text** widget, enter the text you want to display in the form. On the right, you will see the preview.
+![QGIS Text widget configuration](./qgis-configure-text-widget.jpg "QGIS Text widget configuration")
+
+If you prefer your text to be formatted, you may do so in the **HTML** widget. HTML widget supports these [HTML tags](https://doc.qt.io/qt-6/richtext-html-subset.html#supported-tags)).
+![QGIS HTML widget configuration](./qgis-configure-html-widget.jpg "QGIS HTML widget configuration")
+
+...and this is how the Text and HTML widgets look like in the form in QGIS (left) and in the mobile app (right).
+![](./qgis-form-text-html.jpg)
+
+### Using expressions in Text and HTML widgets
+Expressions and variables can be used both in the Text and the HTML widget.
+
+![QGIS Configure Text Widget Expression Builder](./qgis-text-widget-expression.gif "QGIS Configure Text Widget Expression Builder")
+
+1. When configuring the Text or HTML widget, click on the **Expression Builder** button
+2. Enter the expression that will be used in your text and click **OK**.
+
+   Field values can be selected from the **Fields and Values** list. There are other variables and expressions that can be used.
+
+3. Click on the **+** button to add the expression to the text.
+
+   Here, we configured the widget with this text: `Make sure the number plate [% "SPZ" %] is visible in the photo.` 
```

Comment:
`SPZ` is slovak abbreviation (statna poznavacia znacka) - maybe it would be better to name the field "Number Plate" or "Vehicle Registration Plate"

Reply:
Yeah, I wanted to use an existing [project](https://app.merginmaps.com/projects/documentation/form_cascade/tree) from the docs - maybe I could change it also in the project?

Reply:
either that or if it is already in our project feel free to keep it - up to you